pkg/testing/v1alpha1/service.go

```diff
@@ -285,6 +285,19 @@ func WithConfigAnnotations(annotations map[string]string) ServiceOption {
 	}
 }
 
+// WithConfigLabels assigns config labels to a service
+func WithConfigLabels(labels map[string]string) ServiceOption {
+	return func(service *v1alpha1.Service) {
+		if service.Spec.DeprecatedRunLatest != nil {
```

Comment:
Can we use non-deprecated construct here?

Reply:
Sorry if I misunderstood. But non-deprecated construct (I assume that you meant `service.Spec.ConfigurationSpec.Template`) is used at L#295 https://github.com/knative/serving/pull/5807/files#diff-408f318e38522d1eece22893abb0f759R295, if `service.Spec.DeprecatedRunLatest` is `nil`.

Reply:
Hm, I don't know if we even need the other case.
Anyway.